example/ext_ops.yaml

```diff
@@ -112,7 +112,7 @@ spec:
             value: "zk-operator"
 
 ---
-apiVersion: v1beta1
+apiVersion: v1
```

Comment:
👍 I ran into this being a problem when setting up locally. going to v1 on k8s 1.14.3 worked for me too.

Reply:
Yeah thought I would just include those fixes in my big PR, as those files are auto-generated, so I needed to fix the source of those values.

Thanks for pointing it out!